modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -205,6 +205,71 @@
 					title: editor.lang.link.title,
 				});
 
+				/**
+				 * Spacing options are hardcoded to 1.0x, 1.5x, 2.0x, 3.0x, 4.0x and 5.0x,
+				 * so that an empty class, “mt-1 mb-1” class, up to “mt-5 mb-5” are applied respectively.
+				 * These classes define styles via the --spacer-* variables, associated with tokens “Spacer *”,
+				 * and so forth. Classic stylebook defines 11 spacing tokens (--spacer-0 to --spacer-10)
+				 */
+				/**
+				 * TODO:
+				 * 1. Find a way to apply a style to a selection :check:
+				 * 2. Try to host the state of the selected spacing in RichCombo title(Preserve title with the current selection on RichCombo)
+				 */
+				editor.ui.addRichCombo('LineSpacing', {
```

Comment:
@diegonvs did you check the [Figma](https://www.figma.com/file/RiqHAWbLTuDYWjgNgcXWkg/lexi-1084---balloon-toolbar?node-id=164%3A40045) document.
I don't think this can be a `richCombo` based plugin, we need an icon and a label.

Reply:
Hey @diegonvs the [official tutorial](https://ckeditor.com/docs/ckeditor4/latest/guide/plugin_sdk_intro.html) gives a pretty good overview of the plugin "SDK" (I have no idea why they're calling this an SDK but :shrug:), the only downside is that it's a bit basic, and not really enforced: each  CKEditor plugin I've looked at is slightly different. 

That said, I'm sure we can come up with something (`alloyeditor` actually has a very similar component, but it's wrapped in React HOCs which might make it look complex to understand). 

I'd be OK in using the `richCombo` in this first iteration just to have something working - so I think it's just a matter of keeping it as simple as possible for now - seems you're on the right track.